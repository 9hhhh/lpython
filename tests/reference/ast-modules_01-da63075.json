{
    "basename": "ast-modules_01-da63075",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/modules_01.f90",
    "infile_hash": "697383eb5d15cbcf71af84f8421bbe2d4e7a7d094c24b90d5b0c4678",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-modules_01-da63075.stdout",
    "stdout_hash": "b06dc35037f80e5f4fdb0bf3c30e7eb0ac1f186192d13fb9bfb21ee9",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}