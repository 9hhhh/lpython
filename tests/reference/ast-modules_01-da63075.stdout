(TranslationUnit [(Module a [] [] [(Subroutine b [] [] [] [(Print () [(Str "b()")])] [])]) (Program modules_01 [] [] [(SubroutineCall b [])] [])])
